Doc/library/functions.rst

```diff
@@ -893,8 +893,8 @@ are always available.  They are listed here in alphabetical order.
            int(x, base=10)
 
    Return an integer object constructed from a number or string *x*, or return
-   ``0`` if no arguments are given.  If *x* defines :meth:`__int__`,
-   ``int(x)`` returns ``x.__int__()``.  If *x* defines :meth:`__index__`,
+   ``0`` if no arguments are given.  If *x* defines :meth:`~object.__int__`,
+   ``int(x)`` returns ``x.__int__()``.  If *x* defines :meth:`~object.__index__`,
    it returns ``x.__index__()``.  If *x* defines :meth:`__trunc__`,
```

Comment:
Could you also make the same change for references to `__trunc__`?

Reply:
@AlexWaygood Thanks for reviewing. Fixed it.